README.md

```diff
@@ -9,37 +9,47 @@ The plan is to make an oauth2 compatible auth solution with support for the foll
 - Magic link
 - Code
 
-After having been burnt by several freemium model auth solutions that get too expensive once you start growing the user base I wanted a solution where you can get started with a hosted solution but also easily can eject and host it yourself straight on Cloudflare.
+After having been burnt by several freemium model auth solutions that get too expensive once you start growing the user base I wanted a solution where you can get started with a hosted solution but also easily can eject and host it yourself straight.
+
+This service can be hosted on-premise in your own cloud or on a serverless platform such as Cloudflare workers. It uses adapters for different storage providers, but by default it uses SQL storage such as Postgres, MySql or SQLite.
 
 It's based on the following tech stack
 
-- Cloudflare workers (with TSOA)
-- MailChannels for password reset and code/magic links
+- Typescript with the hono router
+- MailChannels or Mailgun for password reset and code/magic links
 
 It contains a hosted UI for the login forms that can be easily styled or modified using liquid templates. Or you can roll your own UI and just use the API.
 
 ## Auth0 compatibility
 
-The goal is to keep the API compatible with the auth0 API so that all the client libraries from auth0 work. This way there's no need to maintain a separate set of client libraries and it makes the migration from auth0 very easy.
+The goal is to keep the API compatible with the auth0 API so that all the client libraries from auth0 work and any exising intergrations with the management API keeps working. This way there's no need to maintain a separate set of client libraries and it makes the migration from auth0 very easy.
 
 ## Tokens
 
-The service keeps a set of JWKS keys stored in KV storage that are used to validate tokens. The tokens are rotated once per day using a cron event, with the last two keys available.
+The service keeps a set of JWKS keys stored that are used to validate tokens. The tokens are rotated once per day using a cron event, with the last two keys available.
 
 The tokens are exposed at: `/.well-known/jwks.json`
 
 ## Getting started
 
-- Change the account_id and the name in the wrangler TOML.
-- Run `yarn` to install the dependencies
-- Run `yarn dev` to start a debug session. This builds the paths and runs `wrangler dev`.
+The easiest way to get started is to run a server locally with bun. This will start a server with a Sqlite database.
+
+```bash
+yarn
+bun scritps/init-sqlite.ts
```

Comment:
The mix of bun and yarn is a bit funny.. Think we should have one version only using node/yarn and a separate with bun.